service.go

```diff
@@ -88,7 +87,7 @@ func (s *Service) SearchCerts(clusterID string) (AssetsBundle, error) {
 func (s *Service) SearchCertsForComponent(clusterID, componentName string) (AssetsBundle, error) {
 	// TODO we should also do a list. In case the secrets have already been
 	// created we might miss them with only watching.
-	watcher, err := s.k8sClient.Core().Secrets(api.NamespaceDefault).Watch(apismetav1.ListOptions{
+	watcher, err := s.k8sClient.Core().Secrets("").Watch(apismetav1.ListOptions{
```

Comment:
I would like to not break or nor change current implementations. Lets maybe rather add a new method. We can cleanup the naming later. 

```
SearchCertsInNamespace(namespace, clusterID string) (AssetsBundle, error)
SearchCertsForComponentInNamespace(namespace, clusterID, componentName string) (AssetsBundle, error)
```

WDYT?

Reply:
Ok. will do tomorrow